ansible/roles/http_proxy/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+use_http_proxy: False
+http_proxy_address: '169.38.84.79:3128'
```

Comment:
this is IP is strange to have as a default. port makes sense, but maybe have the proxy machine be specified in the inventory?